Microsoft Windows [Version 10.0.16299.125]
(c) 2017 Microsoft Corporation. All rights reserved.

C:\Users\USER\Desktop\django-env\Scripts>active
'active' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\USER\Desktop\django-env\Scripts>activate

(django-env) C:\Users\USER\Desktop\django-env\Scripts>cd..

(django-env) C:\Users\USER\Desktop\django-env>cd new_django

(django-env) C:\Users\USER\Desktop\django-env\new_django>dir
 Volume in drive C is Windows 10
 Volume Serial Number is 144F-303A

 Directory of C:\Users\USER\Desktop\django-env\new_django

07/15/2018  08:52 PM    <DIR>          .
07/15/2018  08:52 PM    <DIR>          ..
07/15/2018  08:52 PM                 0 db.sqlite3
07/15/2018  08:43 PM               557 manage.py
07/15/2018  08:52 PM    <DIR>          new_django
               2 File(s)            557 bytes
               3 Dir(s)  182,025,293,824 bytes free

(django-env) C:\Users\USER\Desktop\django-env\new_django>python manage.py migrate

             " First have to creat database for this have to migratre manage.py"

Operations to perform:
  Apply all migrations: admin, auth, contenttypes, sessions
Running migrations:
  Applying contenttypes.0001_initial... OK
  Applying auth.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying sessions.0001_initial... OK

(django-env) C:\Users\USER\Desktop\django-env\new_django>python manage.py runserver
Performing system checks...

System check identified no issues (0 silenced).
July 19, 2018 - 22:23:01
Django version 2.0.7, using settings 'new_django.settings'
Starting development server at http://127.0.0.1:8000/


                                          "now creat a admin user"

(django-env) C:\Users\USER\Desktop\django-env\new_django>python manage.py createsuperuser
Username (leave blank to use 'user'): rafeed
Email address: rafeed717@gmail.com
Password:
Password (again):
This password is too common.
This password is entirely numeric.
Password:
Password (again):
Superuser created successfully.

(django-env) C:\Users\USER\Desktop\django-env\new_django>python manage.py runserver
Performing system checks...


